spec/requests/account_suspended_spec.rb

```diff
@@ -0,0 +1,91 @@
+require "spec_helper"
+
+RSpec.describe "When account is enabled", type: :request do
+  before do
+    allow(Rails.configuration).to receive(:feature_flag_govuk_accounts).and_return(true)
+  end
+
+  context "and accounts is returning a 503 error code" do
+    before { stub_request(:get, Services.accounts_api).to_return(status: 503) }
+
+    describe "/login" do
+      it "redirects a user to the accounts 503 error page" do
```

Comment:
I was considering just metaprogramming these as nearly all the tests are the same, but I wasn't sure where we stood on metaprogramming tests as a team?

I could just shove all these routes in an array and iterate over them if folks prefer that?